finding/finding_test.go

```diff
@@ -198,3 +199,88 @@ func Test_FromBytes(t *testing.T) {
 		})
 	}
 }
+
+func TestOutcome_UnmarshalYAML(t *testing.T) {
+	type args struct {
+		n *yaml.Node
+	}
+	tests := []struct { //nolint:govet
+		name    string
+		o       Outcome
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "positive outcome",
+			o:    OutcomePositive,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Positive",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "negative outcome",
+			o:    OutcomeNegative,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Negative",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "NotAvailable outcome",
+			o:    OutcomeNotAvailable,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "NotAvailable",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "NotSupported outcome",
+			o:    OutcomeNotSupported,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "NotSupported",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "Unknown error",
+			o:    OutcomeError,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Error",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "Unknown outcome",
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Unknown",
+				},
+			},
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Any reason not to parallelize? and therefore include a `tt := tt`?

Reply:
👍